dataset_metrics/Dataset_metrics.ipynb

```diff
@@ -445,6 +445,35 @@
     "page_redirect = spark.sql(query).toPandas()\n",
     "page_redirect.to_csv(os.path.join(output_dir, \"Page redirects\"))"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Number of records filtered out"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "query = \"\"\"SELECT wiki, snapshot,\n",
+    "        SUM(CASE WHEN filter_out = False THEN 1 ELSE 0 END ) as \"Final number of records\",\n",
+    "        SUM(CASE WHEN filter_out = True THEN 1 ELSE 0 END ) as \"Number of records filtered out\"\n",
+    "        FROM clarakosi.imagerec_prod\n",
```

Comment:
Note for later: we should template the query and pass a `username` variable here, instead of hardcoding a db.

Not an issue for know.